### DISCUSSION TITLE
What do you think about my Backtotop extension?

### DISCUSSION THREAD

**Discussion by GiovanniSalmeri** [datenstrom/community#805] [Ask a question] [answered]
Hello everybody. I have just published a new tiny extension:

https://github.com/GiovanniSalmeri/yellow-backtotop

It is a zero-configuration extension which simply creates a back-to-top link in all the pages of the site. I have followed almost all the guidelines of the [Nielsen Norman Group](https://www.nngroup.com/articles/back-to-top/) (one <del>a couple of</del> small intentional deviation<del>s</del> only on point<del>s 1 and</del> 7). Hope you find it useful!

Update: now added a variable (in CSS!) that allows to change the amount of scroll after which the link appears: so it is sufficient to set it to 3 in order to comply with the indication of NN/g (but in my experience this value seems too high). Not implemented the behaviour described in point 7: honestly, I find it a bit annoying wherever I encounter it.

**Comment by PetersOtto** [comment, marked as answer]
It works great! Thank you!

**Reply by GiovanniSalmeri** [reply]
I'm glad! 😊
